System/ram-load.5s.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python3
```

Comment:
This is the shebang most plugins have begun adopting to ensure people using python3 from OS X, installed from homebrew on Intel chip, and installed from homebrew on M1 chip can run it without modifying the script.

```suggestion
#!/usr/bin/env -S PATH="${PATH}:/opt/homebrew/bin:/usr/local/bin" python3
```

Reply:
Have updated the shebang as proposed. Thanks for the tip.